lib/Doctrine/ODM/PHPCR/ChildrenCollection.php

```diff
@@ -42,13 +42,13 @@ class ChildrenCollection extends PersistentCollection
     /**
      * Creates a new persistent collection.
      *
-     * @param DocumentManager $dm         The DocumentManager the collection will be associated with.
-     * @param object          $document   The parent document instance
-     * @param string|array    $filter     Filter string or array of filter string
-     * @param int             $fetchDepth Optional fetch depth, -1 to not override.
-     * @param string          $locale     The locale to use during the loading of this collection
+     * @param DocumentManagerInterface $dm  The DocumentManager the collection will be associated with.
```

Comment:
i am not a huge fan of aligning in general (as it pollutes diffs), but i think re-aligning everything to be perfect here is imho really ugly